examples/ruby/greeter_client.rb

```diff
@@ -29,8 +29,12 @@ def main
   user = ARGV.size > 0 ?  ARGV[0] : 'world'
   hostname = ARGV.size > 1 ?  ARGV[1] : 'localhost:50051'
   stub = Helloworld::Greeter::Stub.new(hostname, :this_channel_is_insecure)
-  message = stub.say_hello(Helloworld::HelloRequest.new(name: user)).message
-  p "Greeting: #{message}"
+  begin
+    message = stub.say_hello(Helloworld::HelloRequest.new(name: user)).message
+    p "Greeting: #{message}"
+  rescue GRPC::BadStatus => e
+    abort "ERROR: #{e.code}, #{e.details}"
```

Comment:
nit: please print the exception's message, rather than individual fields.

Otherwise, we're actually missing one field, `debug_error_string`, added in https://github.com/grpc/grpc/pull/22876, which is coming in next release

Reply:
modified